lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -108,6 +113,18 @@ def load_from_data(self, model, course_structure, scores_client, submissions_sco
             module_id=self.location,
         )
 
+    def grade_data_as_unicode(self):
+        """
+        Provides a unicode representation of the scoring
+        data for this subsection. Used for logging.
+        """
+        return u"|total: {0}/{1}|graded{2}/{3}".format(
+            self.all_total.earned,
+            self.all_total.possible,
+            self.graded_total.earned,
+            self.graded_total.possible,
+        )
```

Comment:
Can we add `self.location` and `self.display_name` here?  (And then you can remove `location` from the caller of this method.)